overlay/boomer.nix

```diff
@@ -19,8 +19,8 @@ in stdenv.mkDerivation rec {
   src = fetchFromGitHub {
```

Comment:
Please, change the version above to `unstable-2020-01-14`. We are following the [Nixpkgs package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming):
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23"

Reply:
right sorry, I didn't see it.